src/OutlineManager.js

```diff
@@ -214,13 +214,82 @@ define(function (require, exports, module) {
         }
     }
 
+    /**
+     * Show the Auto-hide place holder and enable its listener for mouse enter.
+     */
+    function showPlaceHolder() {
+        $(".main-view").append("<div id=\"outline-placeholder\"></div>");
+        var $placeHolder = $("#outline-placeholder");
+        var toolbarPx = $("#main-toolbar:visible").width() || 0;
+        $placeHolder.css("width", "20px");
+        $placeHolder.css("right", toolbarPx + "px");
+        $(".content").css("right", $placeHolder.width() + toolbarPx + "px");
+
+        $placeHolder.on("mouseenter", function () {
+            if (prefs.get("enabled")) {
```

Comment:
You are right. After trying several approaches and changing the code a lot, this has remained there by mistake.